modules/supported-log-outputs.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-output-types.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="supported-log-outputs_{context}"]
+= Supported log forwarding outputs
+
+Outputs can be any of the following types:
+
+.Supported log output types
+[cols="5",options="header"]
+|===
+|Output type
+|Protocol
+|Tested with
+|Logging versions
+|Supported collector type
+
+|Elasticsearch v6
+|HTTP 1.1
+|6.8.1, 6.8.23
+|5.6+
+|Fluentd, Vector
+
+|Elasticsearch v7
+|HTTP 1.1
+|7.12.2, 7.17.7, 7.10.1
+|5.6+
+|Fluentd, Vector
+
+|Elasticsearch v8
+|HTTP 1.1
+|8.4.3, 8.6.1
+|5.6+
+|Fluentd ^[1]^, Vector
+
+|Fluent Forward
+|Fluentd forward v1
+|Fluentd 1.14.6, Logstash 7.10.1, Fluentd 1.14.5
+|5.4+
+|Fluentd
+
+|Google Cloud Logging
+|REST over HTTPS
+|Latest
+|5.7+
+|Vector
+
+|HTTP
+|HTTP 1.1
+|Fluentd 1.14.6, Vector 0.21
+|5.7+
+|Fluentd, Vector
+
+|Kafka
+|Kafka 0.11
+|Kafka 2.4.1, 2.7.0, 3.3.1
+|5.4+
+|Fluentd, Vector
+
+|Loki
+|REST over HTTP and HTTPS
+|2.3.0, 2.5.0, 2.7, 2.2.1
+|5.4+
+|Fluentd, Vector
+
+|Splunk
+|HEC
+|8.2.9, 9.0.0
+|5.7+
+|Vector
+
+|Syslog
+|RFC3164, RFC5424
+|Rsyslog 8.37.0-9.el7, rsyslog-8.39.0
+|5.4+
+|Fluentd, Vector ^[2]^
+
+|Amazon CloudWatch
+|REST over HTTPS
+|Latest
+|5.4+
+|Fluentd, Vector
+|===
+[.small]
+--
+1. Fluentd does not support Elasticsearch 8 in the {logging} version 5.6.2.
+2. Vector does not support Syslog in versions older than the {logging} version 5.7.
```

Comment:
Is this correct? Might was to state what is supported here, rather than what is not. Just a thought.
```suggestion
2. Vector supports Syslog in the {logging} version 5.7 and higher.
```